apps/comps/components/agents-table/index.tsx

```diff
@@ -194,6 +196,21 @@ export const AgentsTable: React.FC<AgentsTableProps> = ({
     }),
   );
 
+  // EigenAI badge status data
+  const { data: eigenBadgeStatuses, isSuccess: isEigenDataLoaded } = useQuery(
+    tanstackClient.eigenai.getBulkBadgeStatuses.queryOptions({
+      input: { competitionId: competition.id },
+    }),
+  );
+
+  // Check if any agent has an active EigenAI badge
```

Comment:
i wonder if putting this in the comps table is the best way to display this information?

i view the competition agents table a "generic" set of metrics associated with the competition. imo it feels a bit odd to have a conditional column that effectively describes an agent's attribute / capability, and not a competition metric. the Eigen badge should be something that belongs on the agent profile page as top-level metadata, similar to the "verified badge".

(maybe this feature is short term and part of partner co-marketing, so it's not something we need to discuss in this PR. but with each partner, this pattern would mean another column and isnt scalable.)